maestro/server/helm/templates/maestro.deployment.yaml

```diff
@@ -54,6 +54,7 @@ spec:
         - --db-user-file=/secrets/db/db.user
         - --db-password-file=/secrets/db/db.password
         - --db-name-file=/secrets/db/db.name
+        - --db-rootcert=
```

Comment:
the default value was file that didn't exist. 
I am leaving this as empty so that the system root cert will be used. 
See https://github.com/Azure/ARO-HCP/pull/998/files#r1888346094